openbb_platform/platform/provider/openbb_provider/standard_models/major_indices_historical.py

```diff
@@ -8,7 +8,7 @@
 from typing import List, Optional, Set, Union
 
 from dateutil import parser
-from pydantic import Field, PositiveFloat, validator
+from pydantic import Field, PositiveFloat, NonNegativeFloat, field_validator
```

Comment:
NonNegativeFloat does not actually fix the underlying issue.  Negative values are OK, and in some cases are the expected values, they should be allowed.

Reply:
Done.